gno.land/pkg/sdk/vm/gas_test.go

```diff
@@ -75,7 +75,7 @@ func TestAddPkgDeliverTx(t *testing.T) {
 	assert.True(t, res.IsOK())
 
 	// NOTE: let's try to keep this bellow 100_000 :)
-	assert.Equal(t, int64(93825), gasDeliver)
+	assert.Equal(t, int64(92825), gasDeliver)
```

Comment:
:)

I'm confused how this went down 🤷‍♂️

Reply:
Because this VMkeeper test does not load the Genesis and does not specify `env.vmk.prmk.SetString(ctx, sysUsersPkgParamPath, "foo")`, namespace support is disabled. Namespace support was costing 1000 gas just to run a gnovm, even if no r/sys/users wasn't existing.

Reply:
TIL